pandas/core/index.py

```diff
@@ -1527,6 +1544,64 @@ def get_indexer(self, target, method=None, limit=None):
 
         return com._ensure_platform_int(indexer)
 
+    def _get_fill_indexer(self, target, method, limit=None):
+        if self.is_monotonic and target.is_monotonic:
```

Comment:
`is_monotonic` is currently an alias for `is_monotonic_increasing`. I'll switch to the later to be more explicit.